build-tools/automation/azure-pipelines-nightly.yaml

```diff
@@ -87,9 +87,9 @@ stages:
           avdApiLevel: 21
           avdAbi: x86
           avdType: default
-        Android23-x86:
+        Android23-x86_64:
           avdApiLevel: 23
-          avdAbi: x86
+          avdAbi: x86_64
```

Comment:
One of the goals of the nightly job was to increase our API/ABI coverage to catch potential issues like this. It seems possible that we have a regression here somewhere, but given that the potential impact is likely quite low I think this is reasonable.